app/src/main/java/com/framgia/oleo/screen/messages/MessagesFragment.kt

```diff
@@ -15,23 +18,39 @@ import com.framgia.oleo.screen.main.MainActivity
 import com.framgia.oleo.utils.OnItemRecyclerViewClick
 import com.framgia.oleo.utils.extension.addFragment
 import com.framgia.oleo.utils.liveData.autoCleared
-import kotlinx.android.synthetic.main.fragment_messages.*
-
-class MessagesFragment : BaseFragment(), OnItemRecyclerViewClick<BoxChat>, SearchView.OnQueryTextListener, View.OnClickListener {
+import kotlinx.android.synthetic.main.fragment_messages.recyclerViewMessages
+import kotlinx.android.synthetic.main.fragment_messages.textSearchMessage
+import kotlinx.android.synthetic.main.fragment_search.searchActionBar
 
+class MessagesFragment : BaseFragment(), OnItemRecyclerViewClick<BoxChat>, View.OnClickListener {
+    private var listener: MessagesFragment.OnSearchListener? = null
     private lateinit var viewModel: MessagesViewModel
     private var binding by autoCleared<FragmentMessagesBinding>()
     private var messagesAdapter = MessagesAdapter()
```

Comment:
sữa lại chỗ này của pull cũ luôn e
```suggestion
    private var messagesAdapter by autoCleared<MessagesAdapter>()
```